databrowser/src/components/quickview/QuickViewCardOverviewContentText.vue

```diff
@@ -1,5 +1,5 @@
 <template>
-  <p class="text-dialog">{{ value }}</p>
+  <p :class="{ 'text-dialog': true, underline }">{{ value }}</p>
```

Comment:
I'm not sure what this is supposed to do, maybe this is a bug?

Reply:
Hi @gappc, this should be fine. Now the component supports an underline property which underlines the text if true.

Reply:
@MatteoBiasi I see, the underline property gave an odd impression but of course it's just `'underline': underline` - my bad ;) 

One improvement would be to apply the `text-dialog` class directly onto the class attribute like `class="text-dialog"`, given that it is always set. The `underline` property would stay as it is and be applied dynamically, like you show it here. While fixing the lint stuff below, it would be very nice if you could apply that change.

Reply:
I did the change. Usually I prefer this annotation to avoid to write the class attribute twice.